### PR TITLE
Qms apis update

### PR DESCRIPTION
**Pull request by muskan-2606**

### CODE REVIEW

File: src/common/db.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ async function connectDB(){
      * Connection URI. Update <username>, <password>, and <your-cluster-url> to reflect your cluster.
      * See https://docs.mongodb.com/ecosystem/drivers/node/ for more details
      */
-    let dbUrl = 'mongodb+srv://qms-db-user:qms-db-pass@qms-dev.ipi25.mongodb.net/qms-dev?retryWrites=true&w=majority';
+
+    let dbUrl = 'process.env.MONGO_URI';
     // let dbUrl = 'mongodb://127.0.0.1:27017/qms-dev';
```
Comment (komekez) [inline src/common/db.ts:12]: Remove this commented line

File: src/common/db.ts [hunk 2]
```diff
@@ -41,4 +43,4 @@ async function listDatabases(dbConncetion: mongoose.Connection){
 export { connectDB }
 
 // the finally block here caused so much trouble, it was closing the connection and I didn't notice
-// https://www.mongodb.com/blog/post/quick-start-nodejs-mongodb--how-to-get-connected-to-your-database
\ No newline at end of file
+// https://www.mongodb.com/blog/post/quick-start-nodejs-mongodb--how-to-get-connected-to-your-database
```
Comment (komekez) [inline src/common/db.ts]: We dont require this kind of comments

File: src/apps/uploadtoS3/router.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+const AWS = require('aws-sdk');
+const multer = require('multer');
+const storage = multer.memoryStorage()
+const upload = multer({storage: storage});
+
+const s3Client = new AWS.S3({
+    accessKeyId: 'your_access_key_id',
+    secretAccessKey: 'your_secret_access_id',
+    region :'ur region'
+});
```
Comment (komekez) [inline src/apps/uploadtoS3/router.ts:12]: Fetch the creds from env

File: src/apps/uploadtoS3/router.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+const uploadParams = {
+         Bucket: 'ur_bucket_name', 
+         Key: '', // pass key
+         Body: null, // pass file body
+};
```
Comment (komekez) [inline src/apps/uploadtoS3/router.ts:18]: pass the parameters

File: src/apps/uploadtoS3/router.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+
+router.post('/api/file/upload', upload.single("file"),(req: { file: { originalname: string; buffer: null; }; },res: { status: (arg0: number) => { (): any; new(): any; json: { (arg0: { error: string; }): void; new(): any; }; }; json: (arg0: { message: string; filename: any; location: any; }) => void; }) => {
```
Comment (komekez) [inline src/apps/uploadtoS3/router.ts:21]: Format the code in proper order and comment about the working of the code

**Review by komekez — CHANGES_REQUESTED** [review]
Discuss and Complete the following changes

**Comment by komekez** [comment]
Need to setup the stage for testing the code

File: src/apps/uploadtoS3/router.ts (hunk 3 above)
Comment (muskan-2606) [reply src/apps/uploadtoS3/router.ts]: done

File: src/common/db.ts (hunk 1 above)
Comment (muskan-2606) [reply src/common/db.ts]: done

**Review by komekez — APPROVED** [review]

**Review by komekez — APPROVED** [review]

**Pull request merged**